tests/e2e/testing-manifests/tkg/tkg.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: run.tanzu.vmware.com/v1alpha1
+kind: TanzuKubernetesCluster
+metadata:
+  name: test-cluster-e2e-script-1
+  namespace: test-gc-e2e-demo-ns
+spec:
+  topology:
+    controlPlane:
+      count: 3
+      class: best-effort-xsmall # vmclass to be used for master(s)
+      storageClass: gc-storage-profile
+    workers:
+      count: 2
+      class: best-effort-xsmall # vmclass to be used for workers(s)
+      storageClass: gc-storage-profile
+  distribution:
+    version: replaceImage 
```

Comment:
how does this work

Reply:
This variable is replaced in the pipeline by getting the tkg images in WCP

Reply:
add a comment
Also test run after the setup in pipeline is complete right? I did not get the flow here...

Reply:
This yaml will be taken from this location by pipeline script and replaceImage will be replaced with the actual tkg image name available in the SDDC.
once the above steps are done, our e2e script will be executed to create TKC.